### DISCUSSION TITLE
Prevent DNS leaks doesn't listen on localhost?

### DISCUSSION THREAD

**Discussion by mvevitsis** [celzero/rethink-app#1617] [Q&A]
I have the option 'prevent DNS leaks' enabled, however when I go to termux and set resolv.conf to 127.0.0.1, nslookup fails. 

I would like to specify the local device/rethink as the DNS server. 

What is the correct address for this interface?

**Comment by ignoramous** [comment]
> What is the correct address for this interface?

As of `v055n`, Rethink (the Android app) sets DNS addresses to `10.111.222.3` / `fd66:f83a:c650::3`.

In termux, running the command `ifconfig` should show interface and DNS addresses for the current active VPN under `tun#`, where `#` is a number (`0`, `1`, `2` etc).

> however when I go to termux and set resolv.conf to 127.0.0.1, nslookup fails

Believe this is an Android limitation, in that it doesn't send loopback (`localhost` / `127.0.0.1`) packets to the active VPN tunnel.
